tests/FunctionalTestCase.php

```diff
@@ -186,8 +186,6 @@ protected function assertCollectionDoesNotExist(string $collectionName, ?string
      * the given name is found, it will be passed to the callback, which may
      * perform additional assertions.
      *
-     * @param string   $collectionName
-     * @param string   $databaseName   Defaults to TestCase::getDatabaseName()
      * @param callable $callback
```

Comment:
Please remove the `@param` line for `$callback` as well. It adds nothing of value.